front/src/services/aleph.ts

```diff
@@ -15,12 +15,9 @@ const ALEPH_GENERAL_CHANNEL = env.ALEPH_GENERAL_CHANNEL;
 
 export class AlephService {
 	constructor(
-		/* eslint-disable-next-line no-unused-vars */
 		private account: ETHAccount,
-		/* eslint-disable-next-line no-unused-vars */
 		private subAccountClient: AuthenticatedAlephHttpClient,
-		// eslint-disable-next-line no-unused-vars
-		private encryptionPrivateKey: PrivateKey,
+		public encryptionPrivateKey: PrivateKey,
```

Comment:
no ESLint errors or warnings by removing these lines ? had it in Vue, strange

Reply:
No because the encryptionPrivateKey is used and is public